Exercise_2.java

```diff
@@ -1,4 +1,4 @@
-class QuickSort 
+class Exercise_2 

```

Comment:
Why did you change the names? It was fine

Reply:
While executing the program in eclipse, I was expected to put the class in its own file. I had to either rename the file to Quicksort.java or the class name. I did the latter.

Reply:
I had actually finished the assignments on time, however, I had not created the pull requests. My bad.